CompilerSource/syntax/syntax.cpp

```diff
@@ -220,26 +226,23 @@ namespace syncheck
             keyword.str("");
           }
         } else {
-          keyword <<c;
+          keyword << c;
         }
       }
       if (!keyword.str().empty()) {
         blacklist.insert(keyword.str());
       }
     }
 
-
     pt pos = 0;
     unsigned mymacroind = 0;
     macro_stack_t mymacrostack;
-    lex.clear(); 
+    lex.clear();
     lex.push_back(token(TT_IMPLICIT_SEMICOLON, ";", superPos, 0, true, false, false, mymacroind));
-          
-    
-    syerr = "No error";
```

Comment:
Nothing, look just a few lines up, I moved it to just before the empty code check.